bedrock/mozorg/middleware.py

```diff
@@ -45,3 +46,17 @@ def process_view(self, request, view, view_args, view_kwargs):
         else:
             f = super(MozorgRequestTimingMiddleware, self)
             f.process_view(request, view, view_args, view_kwargs)
+
+
+class CrossOriginResourceSharingMiddleware(object):
```

Comment:
I spent some time implementing the extra options and I think it starts getting a bit too complex for my taste. None of the requests coming from lorax will be preflighted, and if we eventually want CORS support for preflighted requests on bedrock we can work on integrating a library like django-cors-headers then. For now I think it's overkill. Thoughts?